### DISCUSSION TITLE
Incorrect time saved in event metadata created_at

### DISCUSSION THREAD

**Discussion by Daanooo** [spatie/laravel-event-sourcing#453] [Q&A]
Hi! I am encountering an issue where the created_at in the event metadata does not match up with the actual created at. The event metadata created_at is an hour earlier than the actual created at. 

I have been looking through EloquentStoredEventRepository with a debugger, and in the persist() method on line 83, the created_at is still correct. However, after it passes through the setRawAttributes() method on line 85, the created_at that is being put in the meta_data is incorrect. 

In this screenshot you can see that it's still correct
![image](https://github.com/spatie/laravel-event-sourcing/assets/73757140/0109165b-e161-4a86-a83b-3e0d23aea437)

After passing through the setRawAttributes():
![image](https://github.com/spatie/laravel-event-sourcing/assets/73757140/779ba165-6db9-48b0-9271-5f24b334ffa6)

Our code relies on the event properties to execute tasks in a timely matter, thus it is crucial for this to work correctly. Any ideas are greatly appreciated. Thanks!

**Comment by sebastiandedeyne** [comment]
Which timezone is your application configured in?

**Reply by Daanooo** [reply]
Timezone is in Europe/Amsterdam

**Reply by sebastiandedeyne** [reply]
Then I believe the time is correct; `2024-01-18 12:11` in Amsterdam is the same as `2024-01-18T11:11…Z`. `Z` means "Zulu time" or UTC, which is one hour off from Amsterdam. Date libraries will take that into account when parsing & comparing dates.
